src/www/firewall_rules.php

```diff
@@ -366,7 +366,12 @@ function filter_rule_item_alias_tooltip($alias_name)
                     $("#action").val("del");
                     $("#iform").submit()
                 }
-              }]
+              }],
+              // remove aria-hidden
+              onshown: function(dialogRef){
+                  dialogRef.getModal().removeAttr('aria-hidden');
```

Comment:
I think we should remove this in the bootstrap dialog itself.

https://github.com/opnsense/core/blob/6355a3debd3052526f26c03c029de6de6ed84f2f/src/opnsense/www/js/bootstrap-dialog.js#L392

(By the way, as https://github.com/nakupanda/bootstrap3-dialog seems to be stale, we can update the source on our end.)

I think the original bootstrap 3 examples contained the `aria-hidden` at some time, which might explain why a lot of these forms contain them.